RevenueCatUI/Templates/OnePackageStandardTemplate.swift

```diff
@@ -21,9 +21,7 @@ struct OnePackageStandardTemplate: TemplateViewType {
         VStack(spacing: self.configuration.mode.verticalSpacing) {
             self.scrollableContent
                 .scrollableIfNecessary()
-                .scrollContentBackground(.hidden)
```

Comment:
Was this not needed anymore?

Reply:
It's hidden by default in a scroll view (it's not a `List`). This isn't available in iOS 15, so I just removed it.